cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ParameterUtils.java

```diff
@@ -608,6 +620,7 @@ static ReplicaMovementStrategy getReplicaMovementStrategy(HttpServletRequest req
 
   static List<String> getGoals(HttpServletRequest request) throws UnsupportedEncodingException {
     boolean isKafkaAssignerMode = isKafkaAssignerMode(request);
+    boolean isRebalanceDiskMode = isRebalanceDiskMode(request);
```

Comment:
There seems to be an implicit assumption that if `isKafkaAssignerMode` is `true` then we potentially ignore `isRebalanceDiskMode` -- i.e. when `goals.isEmpty()` -- can we add a sanity check for this case to prevent `isRebalanceDiskMode` to be `true` if `isKafkaAssignerMode` is `true`?